requests/models.py

```diff
@@ -39,7 +40,7 @@
     Callable, Mapping,
     cookielib, urlunparse, urlsplit, urlencode, str, bytes,
     is_py2, chardet, builtin_str, basestring)
-from .compat import json as complexjson
+from .compat import json
```

Comment:
Why did you change this?

Reply:
I'll change it back. It seemed that just `json` would reflect the changes made.